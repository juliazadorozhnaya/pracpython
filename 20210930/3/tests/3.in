1,4,6
2,3,4
5,6,8
1,1,1
2,2,2
3,3,3

