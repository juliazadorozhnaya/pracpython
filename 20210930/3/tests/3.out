[27, 27, 27]
[20, 20, 20]
[41, 41, 41]
